### ISSUE TITLE
Instagram SignIn UI Issue on UWP

### ISSUE THREAD

**Issue by allan-chin** [Clancey/SimpleAuth#96]
@Clancey  - while working on Instagram authentication, I'm suddenly unable to even attempt to log into Instagram, on UWP only. No problem on MacOS. On initial launch, it looks like they're enabled, but as soon as attempt to type something in, nothing gets entered and the "Log In" button gets enabled. Then, if I click in the textbox, all the textboxes get greyed-out (disabled). Do you have idea why this is happening?

Thanks.

![Instagram](https://user-images.githubusercontent.com/30632583/60746808-f3919c00-9f35-11e9-8fae-96b163a7fa8e.png)

**Comment by allan-chin** [comment]
Here's another data point. I played around with this some more, and it looks like I can tab to the textboxes. When I type credentials in, it doesn't appear to be entering, but apparently they are because I get authenticated. When I enter the 1st character of the password, the "Show" displays, but I can continue to type the password over it.

**Comment by allan-chin** [comment]
More data points...

1. I added a Facbook button to the UWP sample and the UI had no issue.
2. I added an Instagram button to your UWP sample and it exhibited the same issue as my app.
3. I copied the InitialUrl into Microsoft Edge and had no issue.
4. I copied the InitialUrl into Microsoft Internet Explorer and the same issue occurs there.
5. Found this, [.](https://social.msdn.microsoft.com/Forums/en-US/ff3631c1-9264-47d7-9906-107a0614cdd7/uwp-how-to-set-webauthenticationbroker-to-use-edge-instead-of-ie?forum=wpdevelop) Apparently, the WebAuthenticationBroker API uses the IE Iframe which doesn't behave well for Instagram.
